articles/GPTs for Business.md

```diff
@@ -0,0 +1,698 @@
+
+(4) **capabilities**, which you can think of as “super powers” you want given to the model - built & maintained exclusively by OpenAI; currently these are limited to: [web browsing](https://openai.com/blog/chatgpt-plugins#:~:text=study%20these%20impacts.-,Browsing,-Alpha), image generation with [DALL-E](https://openai.com/dall-e-3), plus “[code interpreter](https://openai.com/blog/chatgpt-plugins#:~:text=of%20the%20ecosystem.-,Code%20interpreter,-Alpha)” - the ability to spin up a virtual machine to run python code generated by the model which enables mathematics, data analysis & visualization and more. 
+
+>Note: these are different than the inherent capabilities that are always turned on, including: [vision](https://openai.com/research/gpt-4v-system-card), [robust file handling](https://help.openai.com/en/articles/8555545-file-uploads-with-gpts-and-advanced-data-analysis-in-chatgpt), [text to speech](https://openai.com/blog/chatgpt-can-now-see-hear-and-speak#:~:text=on%20all%20platforms.-,Speak%20with%20ChatGPT%20and%20have%20it%20talk%20back,-You%20can%20now)* and [voice](https://openai.com/blog/chatgpt-can-now-see-hear-and-speak#:~:text=voice%20and%20vision.-,Voice,-The%20new%20voice)* (* = only available on Mobile currently)
+
+(5) **[actions](https://platform.openai.com/docs/actions)**, which allow the GPT to connect/integrate to 3rd party systems, so now the model can interact with the real world, on behalf of the user, leveraging their scopes and permissions (with the end user still in the loop, of course)
+
+In addition to the core components, GPTs also allow the GPT builder administrative functions to manage the GPT, including: customization (name, description & avatar, conversation starters) and permissioning (share with: only me, only people in my organization, only people with the link, anyone with paid ChatGPT plan). We won’t go into these ones in detail - but you can checkout our [Help Center documentation](https://help.openai.com/en/articles/8554407-gpts-faq) for more information regarding the administrative & privacy side of GPTs.  
+
+Once a builder has created their GPT, the end user will be able to spin up a conversation with the GPT in the exact same way they are used to interacting with ChatGPT today.
+
+## Getting started building GPTs: Crawl, Walk, Run 
+
+> [!Important]
+>The following is intended to act as a notebook to demonstrate how to build GPTs that enhance value for business users of ChatGPT. Before getting started with these examples, we recommend checking out this guide to help you build and share a basic GPT - as we will skip over many of the basic concepts in our crawl, walk, run examples below:
+>- Crawl: Build a GPT that leverages your internal knowledge in its responses (unstructed data)
+>- Walk: Build a GPT that answers questions about your enterprise data (structured data)
```

Comment:
A simple diagram here could be really useful to understand the control flow / data flow.